### PR TITLE
Add graceful-fs > fs mapping.

### PR DESCRIPTION
**Pull request by dantman**
Also bring NodeRequirer and Require's settings in sync.

### CODE REVIEW

File: NodeRequirer.sublime-settings [hunk 1]
```diff
@@ -21,9 +21,11 @@
         "pouchdb": "PouchDB",
         "prova": "test",
         "es6-promise": "Promise",
+        "native-promise-only": "Promise",
```
Comment (megawac) [inline NodeRequirer.sublime-settings]: :+1:

File: NodeRequirer.sublime-settings [hunk 2]
```diff
@@ -21,9 +21,11 @@
...
         "ractive": "Ractive",
         "ramda": "R",
         "react": "React",
+        "roslib": "ROS",
```
Comment (megawac) [inline NodeRequirer.sublime-settings]: I don't think this one is justified but I dig it

Comment (ganemone) [inline NodeRequirer.sublime-settings]: I'm OK with it.

Comment (dantman) [inline NodeRequirer.sublime-settings]: The only one I added is graceful-fs. This and all the others are mappings which were for some reason defined in `Require.sublime-settings` but not in `NodeRequirer.sublime-settings`.

**Pull request merged**